app/apphandlers/embeddedAppHandler.go

```diff
@@ -112,7 +111,7 @@ func (embedded *EmbeddedApp) StartRudderCore(ctx context.Context, options *app.O
 
 	var tenantRouterDB jobsdb.MultiTenantJobsDB
 	var multitenantStats multitenant.MultiTenantI
-	if config.GetBool("EnableMultitenancy", false) {
+	if misc.UseFairPickup() {
 		tenantRouterDB = &jobsdb.MultiTenantHandleT{HandleT: routerDB}
```

Comment:
shouldn't we rename these as well? e.g. `farpickupRouterDb`, `FairPickupHandle` & `fairPickupStats`

Reply:
[question] what is a fair pickup?

Reply:
in an environment with multiple workspaces (tenants) the fair algorithm makes sure that no workspace is starved when picking up jobs for processing.